template/react-redux-application/src/actions/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as types from '../constants/ActionTypes';
+
+export const updateName = (name: string): Object => {
```

Comment:
Why don't you use the union type (flow)?
It will enable more static type checking in your reducer.